packages/framework-core/src/services/read-model-store.ts

```diff
@@ -182,12 +364,102 @@ export class ReadModelStore {
     return undefined
   }
 
+  public async callFunction<TReadModel extends ReadModelInterface>(
+    projectionMetadata: ProjectionMetadata<EntityInterface, ReadModelInterface>,
+    entity: EntityInterface,
+    migratedReadModel: ReadModelInterface | undefined,
+    readModelID: UUID | undefined
+  ): Promise<ProjectionResult<TReadModel> | undefined> {
```

Comment:
- Are we sure this method should be public or private?
- I would rename it as `callProjectionFunction` since the current name is too generic

Reply:
Made private and renamed in commit 1786d58d0167e3e16ecb19db614fac40ceaebebf.